cypress/integration/tests/manage-studies/create-study.feature

```diff
@@ -6,24 +6,38 @@ Feature: Create Study
     I should be able to create a study
 
 @TestCaseKey=IBP-T511  
-@sanity-test
-@update-install    
- Scenario: Check creation of trial using randomized complete block design with treatment factors
-    Given I am on the Manage Studies page
-    And I created a new study with basic details
-    When I added study settings
+@smoke-test
+Scenario: Check creation of trial using randomized complete block design with treatment factors
+    Given I am on the Manage Studies page of specified program
+    When I created a new study with basic details
+    And I added study settings
     And I browsed germplasm list 
     And I added treatment factors
     And I added environment variables
     And I generated experimental design using Randomized Complete Block design
     Then Design generation should be successful 
-    #Check success message of design generation
-
+    #Check success message of design generation 
```

Comment:
Should we remove this commented line @barbacenasmc , @cheryleigh since already being verified?